### ISSUE TITLE
Consistency check for mandatory fields concerning SORMAS UI and Data Dictionary

### ISSUE THREAD

**Issue by markusmann-vg** [hzi-braunschweig/SORMAS-Project#7648]
### Feature Description

There are fields that in SORMAS are considered to be mandatory but do not have a according entry in the data dictionary.
Example:
![MicrosoftTeams-image](https://user-images.githubusercontent.com/62540178/148972191-39f6774e-88ad-460b-b750-c9145069d44e.png)

This investigation status is marked with a * in the SORMAS UI but in the data dictionary it's not meant to be mandatory.

### Problem Description
This leads to confusion about what is mandatory and what's not-

### Proposed Change
If a field is considered to be mandatory in the UI, it should be also be mentioned as mandatory in the data dictionary.

**Comment by MartinWahnschaffe** [comment]
@markusmann-vg I have checked this and can't confirm it.
The investigation status of the case is marked as a required field int the CaseDataDto since 2018 and the data dictionary correctly reflects this. https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/sormas-api/src/main/resources/doc/SORMAS_Data_Dictionary.xlsx

**Comment by MartinWahnschaffe** [comment]
Got feedback from Stefan Hoffmann, that one of the fields where this is missing, is the Case.caseOrigin field (required in CaseCreateForm).
This would definitely have to be annotated accordingly. I'm note sure how to find out whether other fields are missing as-well.
@MateStrysewske any idea?

**Comment by MateStrysewske** [comment]
@MartinWahnschaffe No, not really.

**Comment by MartinWahnschaffe** [comment]
@FredrikSchaeferVitagroup Could you please have a look at the LabMessage & TestReport and extend the issue description if those two entities have any required fields?

@MateStrysewske Please have a look at the updateded issues description. There are a few discussion points - I'd say that we should have a meeting to decide how to proceed with them. Do you think someone else should be involved?

**Comment by MateStrysewske** [comment]
@MartinWahnschaffe I agree that a meeting would make sense. This is also not only relevant for the data dictionary but for users who are using the ReST API. Would potentially make sense to involve someone who's familiar with lab messages and/or test reports.

**Comment by FredrikSchaeferVitagroup** [comment]
I just checked the [sormas_schema.sql](https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/sormas-backend/src/main/resources/sql/sormas_schema.sql), and there it seems like those columns are required for LabMessage and TestReports:
- LabMessage and TestReport
  - id
  - uuid
  - changedate
  - creationdate
  - sys_period
  - deleted (not explicitly set to NOT NULL but with DEFAULT value)
- LabMessage
  - status
- TestReport
  - labmessage_id

As the more technical attributes are not mentioned for all other intities, I will add `LabMessageDto.status` and `TestReport.labMessage` to the description.
